aptos-move/move-examples/marketplace/readme.md

```diff
@@ -1,6 +1,8 @@
-Exploratory Aptos NFT Marketplace
+[RFC] Aptos NFT Marketplace Standard Proposal
 ------------
 
+NOTE: THIS IS A PROPOSAL AND HAVEN'T BEEN FULLY AUDITED. THESE CONTRACTS ARE FOR COLLECTING FEEDBACKS FROM OUR BUILDERS. ONCE WE ARE CONFIDENT THAT IT IS BENEFICIAL TO TOKEN ECOSYSTEM, WE WILL THEN ADD IT TO ox3::aptos-token PACKAGE
```

Comment:
nit: Can we say "example" instead of "proposal"?

Reply:
plan to go through AIP with this package. currently, just put it in move-examples to show to our builder.